### ISSUE TITLE
🐛 --navigate doesn't work if using lesskey

### ISSUE THREAD

**Issue by PatrickF1** [dandavison/delta#1444]
Hi, I have `--navigate` on but it doesn't work if I load up my lesskey. I don't know how delta makes less search for the •Δ symbols, but I suspect it's through an environment variable that has `--pattern`. And if so, well my lesskey sets a LESS variable which the docs say will override an existing variable. However, I tested passing `--pattern` directly into less on invocation rather than through the LESS environment variable and it still works. So, assuming my guess of how Delta implements --navigate, would it be possible to pass in `--pattern` to less as a command line arg instead of an environment variable so I can use navigation while using a lesskey?

**Comment by PatrickF1** [comment]
Sorry! I did more research and I realized the answer can be found in https://github.com/dandavison/delta/pull/526#issuecomment-790030867. I needed to not set LESSHISTFILE in my lesskey because navigation is implemented by injecting the search pattern into the a new copy of history and passing it into less via the environment variable

**Comment by dandavison** [comment]
Right! Sorry, I meant to reply here earlier -- glad you got there first.
